pkg/controller/kieapp/webconsole/webconsole.go

```diff
@@ -0,0 +1,136 @@
+package webconsole
+
+//go:generate go run -mod=vendor ../defaults/.packr/packr.go
+
+import (
+	"errors"
+	"fmt"
+	"github.com/ghodss/yaml"
+	"github.com/gobuffalo/packr/v2"
+	"github.com/kiegroup/kie-cloud-operator/pkg/controller/kieapp/logs"
+	"github.com/kiegroup/kie-cloud-operator/pkg/controller/kieapp/webconsole/utils"
+	"strconv"
+	"strings"
+
+	consolev1 "github.com/openshift/api/console/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type CustomResourceDefinition struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+}
+
+var log = logs.GetLogger("kieapp.webconsole")
+
+func CreateSampleYAMLs() {
+	log.Info("Loading Console yaml samples.")
+	files, err := loadFilesOnlyWithBox("consoleyamlsamples")
+	if err != nil {
+		log.Error(err, "Failed to load consoleyamlsamples")
+		return
+	}
+
+	resMap, err := applyMultipleWebConsoleYamls(files)
+	if err != nil {
+		log.Error(err, "consoleyamlsamples not successfully applied")
+		return
+	}
+
+	for k, v := range resMap {
+		log.Info("Creating YAML sample. file: % %s ", files[k], v)
+	}
+}
+
+func applyMultipleWebConsoleYamls(yamlString []string) (map[int]string, error) {
+	if yamlString == nil || len(yamlString) == 0 {
+		return nil, errors.New("Empty yaml list")
+	}
+	resMap := make(map[int]string)
+	for idx, yamlStr := range yamlString {
+		_, err := create(yamlStr)
+		if err != nil {
+			resMap[idx] = err.Error()
+		} else {
+			resMap[idx] = "Applied"
+		}
+	}
+	return resMap, nil
+}
+
+func create(yamlStr string) (bool, error) {
```

Comment:
There is a whole lot of complexity here, and I'm not sure why. The operator already has a `writer` that is used throughout the project to create resources.

Reply:
Should've used the reconciler from controller. Modified.
now only webconsole.go is left.